packages/flutter_tools/templates/ffi_plugin/README.md.tmpl

```diff
@@ -0,0 +1,19 @@
+# {{projectName}}
+
+{{description}}
+
+## Getting Started
+
+This project is a starting point for a Flutter 
```

Comment:
Can we expand the readme with a bit more information about how this template works? E.g.:

```
## Getting Started

This project is a starting point for a Flutter 
[FFI plugin](https://docs.flutter.dev/development/platform-integration/c-interop),
a specialized package that includes native code directly invoked with Dart FFI.

## Project stucture

This template uses the following structure:

* `src`: Contains the native source code, and a CmakeFile.txt file for building
  that source code into a dynamic library

* `lib`: Contains the Dart code that defines the API of the plugin, and which
  calls into the native code using `dart:ffi`

* platform folders (`android`, `ios`, `windows`, etc.): Contains the build files
  for including the native code library with the platform application.


## Flutter help

For help getting started with Flutter, view our
[online documentation](https://flutter.dev/docs), which offers tutorials,
samples, guidance on mobile development, and a full API reference.

```